docs/smart-memory-estimation.md

```diff
@@ -22,7 +22,7 @@ Percona XtraBackup performs the backup procedure in two steps:
 
 In the `prepare` phase, Percona XtraBackup checks the server's available free memory and uses that memory up to the limit specified in the [`--use-free-memory-pct`](xtrabackup-option-reference.md#use-free-memory-pct) option to run `--prepare`. Due to backward compatibility, the default value for the `--use-free-memory-pct` option is 0 (zero), which defines the option as disabled. For example, if you set `--use-free-memory-pct=50`, then 50% of the free memory is used to `prepare` a backup.
```

Comment:
Convoluted. 
In the `prepare` phase, Percona XtraBackup checks the server's available free memory and uses that memory up to the limit specified in the [`--use-free-memory-pct`](xtrabackup-option-reference.md#use-free-memory-pct) option to run `--prepare`.

Reply:
Updated